Modules/_threadmodule.c

```diff
@@ -839,6 +839,11 @@ local_traverse(localobject *self, visitproc visit, void *arg)
     return 0;
 }
```

Comment:
Can we add a test to "testcapi"? Even if the tests doesn't answer anything, the sanitizers will detect the problem